### ISSUE TITLE
Make project home page

### ISSUE THREAD

**Issue by madeleineostoja** [peppercornstudio/pollen#41]
And link to existing gitbook docs. Very quick/rough mockup:

![Home](https://user-images.githubusercontent.com/6374876/140912474-7bddb511-2751-4eb8-8af3-1de670e17a55.jpg)

**Comment by ishan-me** [comment]
If I understand this correctly. We want to build a homepage for pollen and link our gitbook there?

**Comment by madeleineostoja** [comment]
Yep! I can hash out the design more if you wanted to work on something!
